roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -24,7 +24,7 @@ server {
   {% if item.value.multisite.enabled | default(false) -%}
     {% if item.value.multisite.subdomains | default(false) -%}
       rewrite ^/(wp-.*.php)$ /wp/$1 last;
-      rewrite ^/(wp-(content|admin|includes)/.*) /wp/$1 last;
+      rewrite ^/(wp-(content|admin|includes).*) /wp/$1 last;
```

Comment:
I added the following line here:

`  rewrite ^/(xmlrpc.php)$ /wp/$1 last;`

or else XML-RPC-dependent stuff (Jetpack authentication, in my case) was not working. Can submit a PR if you want. Thanks for all your hard work!